prince-of-versions/src/main/java/co/infinum/princeofversions/JsonConfigurationParser.java

```diff
@@ -237,8 +241,12 @@ private boolean parseJsonUpdate(JSONObject update, PrinceOfVersionsConfig.Builde
             Iterator<String> metadataIterator = object.keys();
             while (metadataIterator.hasNext()) {
                 String key = metadataIterator.next();
-                Object value = object.get(key);
-                map.put(key, String.valueOf(value));
+                Object value = (!object.isNull(key)) ? object.get(key) : null;
```

Comment:
Is there need to check if field is null or not?
get returns null in case field is set to null.
isNull could be useful in case we don't know if that key exists in json, but we're sure in that as we're iterating through the keys

Reply:
As far as I understood & tested it, it doesn't return the "normal" null, but `JSONObject.NULL`.
The `isNull` then serves to give the "normal" null value to avoid using the `JSONObject.NULL` just for clarity sake.
If I'm mistaken or you think that this is unnecessary and we can use Json null, let me know & I'll fix it 👍

Reply:
You're right, my bad. Thanks for explanation 👍